client/src/App.js

```diff
@@ -1,6 +1,12 @@
+// eslint-disable-next-line no-unused-vars
+/* global chrome */
+
 import { useCallback, useEffect, useState } from 'react';
+import { StyledForm, StyledResult } from './styles';
 
 const App = () => {
+  const url = process.env.REACT_APP_API;
```

Comment:
```suggestion
  const url = process.env.REACT_APP_API || "http://0.0.0.0:5000";
```

Then file `.env.local` will be only needed if we want not std address. Am I correct?

Reply:
Yes, but why .env file then? Previously we wanted to avoid hardcoding api address in many places.

Reply:
In my understanding in above shape the environment variable takes precedence, hence we're able to point frontend to any server address. This environment will be optional (instead of mandatory) setting, fallbacks to `localhost:5000` which most likely is the case (thinking also about docker deployment which will contain all services)

Reply:
As we agreed, we don't want to have local env right now. Please check if everything works.